include/triton/Dialect/TritonGPU/IR/TritonGPUAttrDefs.td

```diff
@@ -530,27 +529,39 @@ For the Threads Per Warp and Values Per Thread level, the linear id distribution
 }
 
 class DistributedEncoding<string name, string attrMnemonic, list<Trait> traits = [],
-                     Dialect dialect = TritonGPU_Dialect> : TritonGPU_Attr<name, attrMnemonic, !listconcat([DistributedEncodingTrait], traits), dialect> {
+                     Dialect dialect = TritonGPU_Dialect>
+  : TritonGPU_Attr<name, attrMnemonic, !listconcat([DistributedEncodingTrait], traits), dialect> {
+
   let description = [{
-Distributed encodings have a layout function that is entirely characterized
-by a d-dimensional tensor L. Note that L doesn't need to have the same shape
+Distributed encodings have a layout function L that is entirely characterized
+by a d-dimensional tensor T. Note that L doesn't need to have the same shape
 (or even the same rank) as the tensor it is encoding.
 
 The layout function \mathcal{L} of this layout is then defined, for an
-index `i` \in R^D, as follows:
+index `i` \in Z^d, as follows:
+
+\mathcal{L}(T)[i_d] = L[(i_d + k_d*T.shape[d]) % L.shape[d]] \forall k_d such as i_d + k_d*T.shape[d] < L.shape[d]
 
-\mathcal{L}(A)[i_d] = L[(i_d + k_d*A.shape[d]) % L.shape[d]] \forall k_d such as i_d + k_d*A.shape[d] < L.shape[d]
+Intuitively, when the tensor dim size T.shape[d] is larger than the layout
+dim size L.shape[d], on that particular dim, we distribute values from the
+tensor to threads mapped in the layout in a "wrapped around" manner, with
+each thread owning multiple values.
+
+OTOH, when the tensor dim size T.shape[d] is smaller than the layout
```

Comment:
What is the rule of using abbreviations in doc?

Reply:
I strive for clarity rather than arbitrary rules like "don't use contractions/abbreviations".  Sometimes abbreviating things can make it easier to communicate (uses less space), sometimes it can make it harder.

Reply:
I think we can just drop the `OTOH` here without impacting the flow too much. Did that.